<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagramPrefs</class>
 <widget class="QDialog" name="DiagramPrefs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diagram Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="stationTypeLabel">
     <property name="text">
      <string>StationType</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="stationNameLabel">
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>StationName</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="viewDataLabel">
     <property name="text">
      <string>View Data</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="DateBoundsLayout">
     <property name="margin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>20</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="fromDateLabel">
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="toDateLabel">
       <property name="text">
        <string>To:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="toDateLine"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="fromDateLine"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="CheckBoxLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ButtonLayout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadFileButton">
       <property name="text">
        <string>Load File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fromDateLine</tabstop>
  <tabstop>toDateLine</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>loadFileButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiagramPrefs</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiagramPrefs</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
